### DISCUSSION TITLE
Help - don't know what im doing?

### DISCUSSION THREAD

**Discussion by Nick534** [jamulussoftware/jamulus#3218] [Technical Support & Help] [answered]
Hi folks, I'm new to Jamulus and a complete technophobe. My missus has downloaded Jamulus for me. I'm running a Behringer Uphoria 404 interface. When logged on, I can hear other musicians but not myself. I can only hear my own instrument by pushing the monitor button which then cancels out the other guys. Am I doing something ridiculously stupid or could it be, that the software hasn't been installed properly. Also, do I need an Asio4all if I'm already running an interface.

**Comment by jujudusud** [comment]
Hi Nick, if your interface is the one of the link:
`https://www.behringer.com/product.html?modelCode=P0BK1`
You have to go download the Windows driver in this page here:
![Screenshot_2024-01-02-17-25-06-676_org mozilla firefox-edit](https://github.com/jamulussoftware/jamulus/assets/12748768/fb922b65-a845-4be0-9238-552b080cb5e9)
And install it.
After that you will be able uninstall asio4all.
After that it should work flawlessly.

**Reply by Nick534** [reply]
The missus downloaded it from where you said for the right machine.  Excuse my ignorance; What is Allez?

**Reply by jujudusud** [reply]
Ok, perfect for the driver. You don't need asio4all.
Allez is a french word, thanks to my automatic orthograph corrector... :-)
In order to ear the sound returned from the computer you have put the mix knob to "PB" (playback):
![Screenshot_2024-01-02-22-19-20-146_org mozilla firefox-edit](https://github.com/jamulussoftware/jamulus/assets/12748768/fd6267cf-1684-4881-833c-57e85fc80076)

**Reply by Nick534** [reply]
Will try again and thank you for taking the time to help👍

**Reply by Nick534** [reply]
Hi mate,I tried your suggestion with monitor button on and off and sadly no success,same problem,ican hear the other guys with monitor pushed in but not myself,release monitor button and I appear and the others vanish. 🤦 Could I be doing something wrong with the Behringer? It's the first time I've used an interface,  computer etc. Mainly acoustic, how about yourself?

**Reply by rdica** [reply]
In PB mode did you have the volume of your channel in the jamulus client turned up as well?
You'd only hear yourself if you were connected to a server.

**Reply by jujudusud** [reply]
Who talk about the monitor botton?
I was talking about the Mix knob turned 100% to PB (right side).

**Reply by Nick534** [reply]
Thanks again,I did try the p.b. but will check client volume thanks

**Comment by QJKX** [comment, marked as answer]
I have that interface.

It should look like this:

![image](https://github.com/jamulussoftware/jamulus/assets/23462605/dbce55c2-ad61-4b94-9f04-37bb53bb4475)



You'll see on the physical Audio Interface the inputs are numbered "Input 1" (on the far left), which here is "In 1".
My guess is the inputs you're plugged into don't match what's configured here.


This device has 4 outputs 1/2/3/4.
The headphones can use 1/2 or 3/4.
The monitor A/B button makes that choice.

With 1/2 it also goes through the Mix knob.
As jujudusud has pointed out, for Jamulus that knob MUST be turned clockwise to "PB"


Much better than the manual is this unofficial [signal path diagram for the Behringer UMC-404HD audio interface](https://github.com/mcnster/umc-404hd-signal-path/blob/master/umc404hd.pdf).

**Reply by Nick534** [reply]
Thank you,think it's sorted now,much appreciation to you guys.

**Comment by Nick534** [comment]
Hi folks can I run a boss gt6 through my behringer umc 404?

**Reply by jujudusud** [reply]
Yes, you can.

**Reply by QJKX** [reply]
Yes, you want it as a line level signal.

Set the gt6 "output select" to line/phones.

Set the audio interface inputs to line level physically - there's an instrument/line button next to each input. Turn down the gain.

connect the output L (mono) to an audio interface input. Optionally connect output R too, if you want stereo. Connect your headphones to the audio interface.


If you chose stereo you should set the jamulus screen the screenshot is of ("audio/network settings) to stereo in/stereo out.

The gt6 will probably sound best if you have some sort of speaker emulation enabled.

connect to an empty server to see if it works.

To get your signal level best set up, while playing guitar at the loudest you will (boost/solo/whatever) turn the gain knobs up on the interface as much as you can without the red "clip" lights on the interface ever coming on.

Adjust how loud you actually want to sound relative to others on the jamulus mixer.
Adjust the overall volume of everyone on the headphone volume knob.

**Reply by Nick534** [reply]
Thanks for the info👍

**Comment by Nick534** [comment]
Thank you. Guitar in to GT6 and one output to interface?

Sent from Yahoo Mail on Android 
 
  On Sun, 14 Jan 2024 at 12:55, Julien ***@***.***> wrote:   


Yes, you can.

**Comment by Nick534** [comment]
Thank you, will try this now.
Nick 

    On Sunday, 14 January 2024 at 13:21:08 GMT, QJKX ***@***.***> wrote:  
 
 


Yes, you want it as a line level signal.

Set the gt6 "output select" to line/phones.

Set the audio interface inputs to line level physically - there's an instrument/line button next to each input. Turn down the gain.

connect the output L (mono) to an audio interface input. Optionally connect output R too, if you want stereo. Connect your headphones to the audio interface.

If you chose stereo you should set the jamulus screen the screenshot is of ("audio/network settings) to stereo in/stereo out.

The gt6 will probably sound best if you have some sort of speaker emulation enabled.

connect to an empty server to see if it works.

To get your signal level best set up, while playing guitar at the loudest you will (boost/solo/whatever) turn the gain knobs up on the interface as much as you can without the red "clip" lights on the interface ever coming on.

Adjust how loud you actually want to sound relative to others on the jamulus mixer.
Adjust the overall volume of everyone on the headphone volume knob.
